public/homepage/scripts/main.js

```diff
@@ -32,17 +32,11 @@ require.config({
 
 // While the user is reading this page, start to cache Bramble's biggest files
 function preloadBramble($) {
-  var brambleHost = $("meta[name='bramble-host']").attr("content");
-  brambleHost = brambleHost.replace(/\/$/, "");
-  [
-    brambleHost + "/dist/styles/brackets.min.css",
-    brambleHost + "/dist/bramble.js",
-    brambleHost + "/dist/main.js",
-    brambleHost + "/dist/thirdparty/thirdparty.min.js"
-  ].forEach(function(url) {
-    // Load and cache files as plain text (don't parse) and ignore results.
-    $.ajax({url: url, dataType: "text"});
-  });
+//Using prefetch for easier cacheing and preloading of resources
+  <link rel="prefetch" href="/dist/styles/brackets.min.css">
```

Comment:
This is HTML, and you're putting it in a JavaScript file.  You can't mix these.  This would have to go in the HTML for the homepage, see https://github.com/mozilla/thimble.mozilla.org/blob/master/views/homepage/index.html.

Next issue: you need to fix the URLs you are loading, as these are going to try and load from the Thimble server.